gem/views.py

```diff
@@ -280,6 +281,32 @@ def get(self, request, comment_pk):
         })
 
 
+class KaiOSManifestView(View):
+    def get(self, request):
```

Comment:
Almost.. you'll need to also change the headers as stipulated in the docs, so it might not necessarily be this method to use.

Reply:
hmmm, ok